content_scripts/inject_script.coffee

```diff
@@ -0,0 +1,16 @@
+injectScripts = ["pages/addEventListener_hook.js"]
+
+fetchFileContents = (extensionFileName) ->
+  req = new XMLHttpRequest()
+  req.open("GET", chrome.runtime.getURL(extensionFileName), false) # false => synchronous
+  req.send()
+  req.responseText
```

Comment:
Should we not re-use [this](https://github.com/philc/vimium/blob/master/content_scripts/ui_component.coffee#L124)?

(Probably moving it to `dom_utils.coffee`.)